docs/troubleshoot.md

```diff
@@ -6,92 +6,93 @@ indent: true
 ---
 # Troubleshooting
 
-* [Crossplane Logs](#crossplane-logs)
 * [Resource Status and Conditions](#resource-status-and-conditions)
+* [Crossplane Logs](#crossplane-logs)
 * [Pausing Crossplane](#pausing-crossplane)
 * [Deleting a Resource Hangs](#deleting-a-resource-hangs)
 
-## Crossplane Logs
+## Resource Status and Conditions
 
-The first place to look to get more information or investigate a failure would be in the Crossplane pod logs, which should be running in the `crossplane-system` namespace.
-To get the current Crossplane logs, run the following:
+Most Crossplane resources have a `status` section that can represent the current
+state of that particular resource. Running `kubectl describe` against a
+Crossplane resource will frequently give insightful information about its
+condition.For example, to determine the status of a MySQLInstance resource
+claim, run:
 
-```console
-kubectl -n crossplane-system logs -lapp=crossplane
+```shell
+kubectl -n app-project1-dev describe mysqlinstance mysql-claim
 ```
 
-## Resource Status and Conditions
-
-All of the objects that represent managed resources such as databases, clusters, etc. have a `status` section that can give good insight into the current state of that particular object.
-In general, simply getting the `yaml` output of a Crossplane object will give insightful information about its condition:
+This should produce output that includes:
 
 ```console
-kubectl get <resource-type> -o yaml
+Status:
+  Conditions:
+    Last Transition Time:  2019-09-16T13:46:42Z
+    Reason:                Managed claim is waiting for managed resource to become bindable
+    Status:                False
+    Type:                  Ready
+    Last Transition Time:  2019-09-16T13:46:42Z
+    Reason:                Successfully reconciled managed resource
+    Status:                True
+    Type:                  Synced
 ```
 
-For example, to get complete information about an Azure AKS cluster object, the following command will generate the below sample (truncated) output:
+Most Crossplane resources set exactly two condition types; `Ready` and `Synced`.
```

Comment:
should `exactly` be rephrased as `at least`?

Reply:
I considered that, but I think it is actually exactly in all cases. At least would leave us space to add more in the future, but exactly is more accurate today.